pallets/api/src/fungibles/benchmarking.rs

```diff
@@ -103,5 +106,69 @@ mod benchmarks {
 		Ok(())
 	}
 
+	#[benchmark]
+	// Storage: `Assets`
+	fn total_supply() {
+		#[block]
+		{
+			Pallet::<T>::read(Read::TotalSupply(TokenIdOf::<T>::zero()));
+		}
+	}
+	#[benchmark]
+	// Storage: `AssetAccount`
+	fn balance_of() {
+		let owner: AccountIdOf<T> = account("Alice", 0, SEED);
```

Comment:
Minor, but `owner: account("Alice", 0, SEED)` inline below might be simpler. It conveys the same in a more compact style. Same with the others benchmarks.

Reply:
see latest commit